roles/ruby_s/tasks/cleanup.yml

```diff
@@ -1,25 +1,28 @@
 ---
-- name: ruby | uninstall global bundler
+- name: ruby_s | uninstall global bundler
   community.general.gem:
     name: bundler
     state: absent
-  changed_when: false
 
-- name: ruby | remove installed ruby
+- name: ruby_s | check for apt-installed rubies
+  ansible.builtin.package_facts:
+    manager: apt
+  register: all_packages
```

Comment:
It doesn't look like we're using this value? Is this to aid debugging? Maybe it should be removed, or commented.

Reply:
the value gets used in line 14

Reply:
oh, I see what you mean, we don't need to register the var because it becomes part of ansible_facts . . . I'll make an issue for this